pkg/setting/setting_smtp.go

```diff
@@ -18,8 +21,14 @@ type SmtpSettings struct {
 	SendWelcomeEmailOnSignUp bool
 	TemplatesPatterns        []string
 	ContentTypes             []string
+	Theme                    string
 }
 
+const (
```

Comment:
AFAIK we're not going to support dark and light themes. Should we remove these options, then?

Reply:
Yep, should probably clean that up – good shout!